docs/Downloaders/SABnzbd/Basic-Setup.md

```diff
@@ -80,17 +83,18 @@ Covered and fully explained in [SABnzbd - Paths and Categories](/Downloaders/SAB
 
 ## Switches
 
-[Sabnzbd Documentation](https://sabnzbd.org/wiki/configuration/4.0/switches){:target="_blank" rel="noopener noreferrer"}
+[Sabnzbd Documentation](https://sabnzbd.org/wiki/configuration/4.3/switches){:target="_blank" rel="noopener noreferrer"}
 
 ### Queue
 
 `Settings` => `Switches` => `Queue`
 ![!Switches: Queue](images/sabnzbd-switches-queue.png)
 
+1. If you have really good indexers that get nzb from post not generated then some people might want Propagation delay  set to 5 min. so your not trying to grab a nzb 0 mins from posting and if your not getting it from the same server the poster used you might wrongly have article missing since it hasn't propagated to your server.. or if you use a reseller it may take them longer to get it from their upstream.
```

Comment:
```suggestion
1. If you have good indexers that gets the nzb from the post, not generated, then you may want the Propagation delay set to 5 minutes (so you're not trying to grab an nzb right at posting). If you're not getting it from the same server as the poster used, you might wrongly have articles missing (since it hasn't necessarily propagated to your server yet) or if you use a reseller it may take them longer for them to get it from their upstream.